packages/integration-tests/src/package_manager_sanity_checks.test.ts

```diff
@@ -130,4 +133,17 @@ void describe('getting started happy path', async () => {
 
     assert.ok(clientConfigStats.isFile());
   });
+
+  void it('creates new project and deploy them without an error', async () => {
+    if (packageManager === 'pnpm' && process.platform === 'win32') {
+      try {
+        await execa('pnpm', ['create', 'amplify@beta', '--yes'], {
+          cwd: tempDir,
+        });
+      } catch (error) {
+        const errorMessage = error instanceof Error ? error.message : '';
+        assert.match(errorMessage, /Amplify does not support PNPM on Windows./);
+      }
+    }
```

Comment:
If we are expecting an error, we should use `assert.rejects`

Reply:
`assert.rejects` doesn't work here because the error is from a child process 😢